src/code42cli/logger.py

```diff
@@ -40,6 +62,36 @@ def get_logger_for_stdout(name_suffix="main", formatter=None):
     return logger
 
 
+def get_logger_for_server(hostname, protocol, output_format):
```

Comment:
Figured out so far that RAW-JSON works perfectly but none of the other formats do, which tells me this is an error with the Formatter that is causing this not to function.... will keep digging